internal/pkg/gateway/api.go

```diff
@@ -70,19 +85,49 @@ func (gs *Server) Endorse(ctx context.Context, proposedTransaction *pb.ProposedT
 		return nil, fmt.Errorf("failed to extract value from response payload: %w", err)
 	}
 
-	preparedTxn := &pb.PreparedTransaction{
+	preparedTxn := &gp.PreparedTransaction{
 		TxId:      proposedTransaction.TxId,
-		ChannelId: proposedTransaction.ChannelId,
+		ChannelId: channel,
 		Response:  retVal,
 		Envelope:  env,
 	}
 	return preparedTxn, nil
 }
 
 // Submit will send the signed transaction to the ordering service.  The output stream will close
-// once the transaction is committed on a sufficient number of peers according to a defined policy.
-func (gs *Server) Submit(txn *pb.PreparedTransaction, cs pb.Gateway_SubmitServer) error {
-	// not yet implemented in embedded gateway
+// once the transaction is committed on a sufficient number of remoteEndorsers according to a defined policy.
+func (gs *Server) Submit(txn *gp.PreparedTransaction, cs gp.Gateway_SubmitServer) error {
+	if txn == nil {
+		return fmt.Errorf("a signed prepared transaction is required")
+	}
+	if cs == nil {
+		return fmt.Errorf("a submit server is required")
+	}
+	orderers, err := gs.registry.orderers(txn.ChannelId)
+	if err != nil {
+		return err
+	}
+
+	if len(orderers) == 0 {
+		return fmt.Errorf("no broadcastClients discovered")
+	}
+
+	// send to first orderer for now
+	logger.Info("Submitting txn to orderer")
```

Comment:
We'll want to revisit this. An info message with no context isn't particularly helpful to anyone.